doc/man3/EVP_PKEY_check.pod

```diff
@@ -38,11 +38,6 @@ It is not necessary to call these functions after locally calling an approved ke
 generation method, but may be required for assurance purposes when receiving
 keys from a third party.
 
-In OpenSSL an EVP_PKEY structure containing a private key also contains the
-public key components and parameters (if any). An OpenSSL private key is
-equivalent to what some libraries call a "key pair". A private key can be used
-in functions which require the use of a public key or parameters.
-
```

Comment:
Does it make sense adding some text to HISTORY about this change?

Reply:
Hmmm, perhaps?  It's not like any type or function has been *added* or *removed*, which is what we usually mention in the history section.  If you have an idea what to actually write in the history section, I'm all ears.

Reply:
I'd suggest mentioning that the presence of both public and private key for private keys was implied before 3.0.

Reply:
Ah, good point

Reply:
Done